src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -47,6 +48,13 @@
 public class HologramProjector extends SlimefunItem implements HologramOwner {
 
     private static final String OFFSET_PARAMETER = "offset";
+    private static final String OFFSET_PARAMETER_X = "offset-x";
+    private static final String OFFSET_PARAMETER_Z = "offset-z";
+    private static final double MAX_HORIZONTAL_OFFSET = 32.0D; // Up to 2 chunks away in any direction horizontally
```

Comment:
I feel like this might be too big, how this works with chunkloading already worries me, allowing it to go this far only worsens that

Reply:
If getting the armorstand causes the chunk to load, which I think it does (could be totally wrong tho), this would now act as a chunk loader

Reply:
Alr in that case it should probably be fine 👍